client/core/trade.go

```diff
@@ -705,14 +714,16 @@ func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 // shouldBeginFindRedemption will be true if we are the Taker on this match,
 // we've broadcasted a swap, our swap has gotten the required confs, we've
 // not been notified of Maker's redeem and we've not refunded our swap.
+//
+// This method accesses match fields and MUST be called with the trackedTrade
+// mutex lock held for reads.
 func (t *trackedTrade) shouldBeginFindRedemption(match *matchTracker) bool {
-	proof := &match.MetaData.Proof
+	dbMatch, _, proof, _ := match.parts()
 	swapCoinID := proof.TakerSwap
-	if match.failErr != nil || match.Match.Side != order.Taker || len(swapCoinID) == 0 ||
-		len(proof.MakerRedeem) > 0 || len(proof.RefundCoin) > 0 {
-		log.Tracef(
+	if dbMatch.Side != order.Taker || len(swapCoinID) == 0 || len(proof.MakerRedeem) > 0 || len(proof.RefundCoin) > 0 {
+		log.Debugf(
```

Comment:
Tracef please as I think this will spam a lot.  When testing for this with jy-p and gang we'll put logs to TRC level.

Reply:
Thought so too, but noticed similar checks for isSwappable and isRedeemable used Debugf. Reverting to Tracef for all those checks.